pkg/api/customization/cluster/actions_cis.go

```diff
@@ -62,11 +64,35 @@ func (a ActionHandler) runCisScan(actionName string, action *types.Action, apiCo
 		return httperror.WrapAPIError(err, httperror.ClusterUnavailable,
 			fmt.Sprintf("cluster not ready"))
 	}
+	if cluster.Spec.WindowsPreferedCluster {
+		return httperror.WrapAPIError(err, httperror.InvalidAction,
+			fmt.Sprintf("cannot run scan on a windows cluster"))
+	}
 	if _, ok := cluster.Annotations[v3.RunCisScanAnnotation]; ok {
 		return httperror.WrapAPIError(err, httperror.Conflict,
 			fmt.Sprintf("CIS scan already running on cluster"))
 	}
 
+	if cisScanConfig.OverrideBenchmarkVersion != "" {
+		versions, err := a.CisBenchmarkVersionLister.List(namespace.GlobalNamespace, labels.Everything())
```

Comment:
we could directly search for OverrideBenchmarkVersion as name here?

Reply:
👍